.github/workflows/flowzone.yml

```diff
@@ -301,33 +301,37 @@ jobs:
       versionbot: ${{ steps.versionbot_checkout.outcome }}
 
     steps:
-      - name: Checkout versioned source
-        id: versionbot_checkout
-        continue-on-error: true
-        uses: actions/checkout@v3
-        with:
-          ref: versionbot/pr/${{ github.event.pull_request.number }}
-          fetch-depth: 2
-          token: ${{ secrets.FLOWZONE_TOKEN }}
-
-      - name: Assert that versioned source is up-to-date
-        if: ${{ steps.versionbot_checkout.outcome == 'success' }}
-        run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
+      # - name: Checkout versioned source
+      #   id: versionbot_checkout
+      #   continue-on-error: true
+      #   uses: actions/checkout@v3
+      #   with:
+      #     ref: versionbot/pr/${{ github.event.pull_request.number }}
+      #     fetch-depth: 2
+      #     token: ${{ secrets.FLOWZONE_TOKEN }}
 
       # Fallback to unversioned source if versionbot branch does not exist
       - name: Checkout original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
+        # if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1
           token: ${{ secrets.FLOWZONE_TOKEN }}
 
+      # - name: Assert that versioned source is up-to-date
+      #   # if: ${{ steps.versionbot_checkout.outcome == 'success' }}
+      #   run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
+
       # Run balena-versionist in Github actions when the versionbot branch could not be checked out
       - name: Version original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
+        # if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         run: |
           npm install -g balena-versionist versionist
           balena-versionist
+          git status
+          VERSION="$(head -n1 VERSION || jq -r '.version' package.json)"
+          git add --all
+          git -c "user.name=Versionist" -c "user.email=versionist@balena.io" commit --allow-empty -m "${VERSION}"
```

Comment:
Are these credentials associated with any real account that might get mapped when looking at the commit history? Not sure it matters, mostly wondering if these are fully made up or not.

Reply:
We also need to update all the places where we check if author contains 'versionbot'

Reply:
@klutchell, I suggest we use the name `versionbot` with the email address `versionbot@versionbot.product-os.io`.  The versionbot.product-os.io domain is set up in Mailgun and we can forward emails to a webhook or another email address if needed.